### DISCUSSION TITLE
add LDAP_ATTRIBUTE_FOR_MAIL in LDAP configuration

### DISCUSSION THREAD

**Discussion by lich72** [open-webui/open-webui#8224] [Ideas]
# Feature Request

**Is your feature request related to a problem? Please describe.**
In our LDAP settings, the mail value is not from "mail" but another attribute. we can not get mail from LDAP

**Describe the solution you'd like**
add LDAP_ATTRIBUTE_FOR_MAIL in LDAP configuration

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by steelcg** [comment]
PR for this topic: #8416
